index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    
+</body>
+</html>
```

Comment:
- [ ] As stated in the project in your html file that includes a header with the text "Hello Microverse!". Use h1 tag for text "Hello Microverse!" that inside a body tag Just like this :- 
```
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <link rel="stylesheet" type="text/css" href="Style.css">
        <title>Hello Microverse</title>
        </head>
        <body>
            <h1 class="header">Hello Microverse!</h1>
        </body>
        </html> 
```